ajit_2:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  rear_right_wheel_position_controller:
    type: position_controllers/JointPositionController
    joint: rear_right_wheel
    pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}
    # set: {1, 1, 1, -0.041193, -0.455106, 0.215851}

  front_right_wheel_position_controller:
   type: position_controllers/JointPositionController
   joint: front_right_wheel
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}
   # set: {1, 1, 1, -0.041193, -0.455106, 0.215851}

  front_left_wheel_position_controller:
   type: position_controllers/JointPositionController
   joint: front_left_wheel
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}
   # set: {1, 1, 1, -0.041193, -0.455106, 0.215851}

  rear_left_wheel_position_controller:
   type: position_controllers/JointPositionController
   joint: rear_left_wheel
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}
   # set: {1, 1, 1, -0.041193, -0.455106, 0.215851}

  right_shoulder_joint_position_controller:
   type: position_controllers/JointPositionController
   joint: right_shoulder_joint
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}

  left_shoulder_joint_position_controller:
   type: position_controllers/JointPositionController
   joint: left_shoulder_joint
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}

  right_shoulder_position_controller:
   type: position_controllers/JointPositionController
   joint: right_shoulder
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}

  left_shoulder_position_controller:
   type: position_controllers/JointPositionController
   joint: left_shoulder
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}

  right_upper_arm_position_controller:
   type: position_controllers/JointPositionController
   joint: right_upper_arm
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}

  left_upper_arm_position_controller:
   type: position_controllers/JointPositionController
   joint: left_upper_arm
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}

  right_lower_arm_position_controller:
   type: position_controllers/JointPositionController
   joint: right_lower_arm
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}

  left_lower_arm_position_controller:
   type: position_controllers/JointPositionController
   joint: left_lower_arm
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}

  right_hand_position_controller:
   type: position_controllers/JointPositionController
   joint: right_hand
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}

  left_hand_position_controller:
   type: position_controllers/JointPositionController
   joint: left_hand
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}

  head_position_controller:
   type: position_controllers/JointPositionController
   joint: head
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}

  neck_position_controller:
   type: position_controllers/JointPositionController
   joint: neck
   pid: {p: 1.0, i: 1.0, d: 1.0, i_clamp_min: -10.0, i_clamp_max: 10.0}
